src/store/popularCategoriesSlice/slice.ts

```diff
@@ -0,0 +1,58 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { LOAD_STATUSES } from "../../constants";
+import { Category, getPopularCategories, Good } from "api/api";
+
+export interface State {
+  popularCategories: { category: Category; items: Good[] }[];
+  loadStatus: LOAD_STATUSES;
+}
+
+const initialState: State = {
+  popularCategories: [
+    {
+      items: [
```

Comment:
массивы пустыми объектами не забиваем, делаем просто пустой массив
```suggestion
      items: [],
```